pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Output all lines where column with field_name is numerically equal to value:
```

Comment:
I think I prefer print because display is even more suggestive of the output appearing in the terminal - even though one may end up piping it into a file (I often do) 

That's why I initially used `output` which is more neutral as to how that output is used/displayed/printed

Reply:
I committed @navarroaxel's version but don't feel strongly about this

Reply:
We use display and print depending on the style of the command, however normally both can be used.

However I think print fits better here so I agree with you.